src/components/Footer.jsx

```diff
@@ -1,92 +1,93 @@
-import React,{useState} from "react";
+import React, { useState } from 'react';
```

Comment:
What lint are you using ? Understand eslint and how to leverage it; linting has messed up.